templates/ces_client/about.html

```diff
@@ -0,0 +1,76 @@
+{% extends "base.html" %}
+{% load static %}
+{% load ces_client_extras %}
+{% block title %}About{% endblock %}
+
+{% block extra_css %}
+    <link rel="stylesheet" href="{% static 'css/botui.min.css' %}" />
+    <link rel="stylesheet" href="{% static 'css/botui-theme-default.css' %}" />
+{% endblock %}
+
+{% block content %}
+<div class='row'>
+    <div class='col-10 offset-1 mt-5 mb-3 pb-3'>
+        <div class='row mb-3'>
+            <div class='col-lg-2 pr-0'>
+                <a class="img-fluid float-left mr-0" href="{% url 'index' %}"><img src="{% static 'images/CESlogo.svg' %}" width='100'></a>
+            </div>
+            <div class='col'>
+                <h3 class='mt-4'>Connect to End Homelessness</h3>
+                <h4>A unified community response</h4>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class='row'>
+    <div class='col-10 offset-1 mb-3 pb-3'>
+        <h5 class="font-weight-bold">About Coordinated Entry</h5>
+        <p>Coordinated Entry is the front door to connect people facing homelessness to housing and supports in an accessible, equitable, transparent manner. This allows all partners to collaborate to improve access and outcomes for people without permanent housing while delivering trauma informed services.</p>
+        <p>People facing homelessness are asked to complete a Housing Assessment and linked to housing, if eligible, as opportunities become available.</p>
+
+        <h5 class='mt-5 font-weight-bold'>Shelter Access in Cook County Suburbs</h5>
+        <p>View shelter options and next steps <a href="http://www.suburbancook.org/emergency" target="_blank">by clicking here</a>.</p>
+
+        <h5 class='mt-5 font-weight-bold'>Shelter Access In Chicago</h5>
+        <p>Please visit any hospital emergency room or police station in Chicago, or call 311 and request transportation to shelter.</p>
+        <div class='row'>
+            <div class='col'>
```

Comment:
I was imagining something like this, with the icon centered and larger:
<img width="459" alt="Screen Shot 2019-07-03 at 10 20 02 AM" src="https://user-images.githubusercontent.com/6961258/60604007-6eaf5280-9d7c-11e9-8588-5262d08b96e7.png">

Having it in line with the text is difficult to read.

Reply:
Got it! Thanks for the image, too.